Monika After Story/game/script-topics.rpy

```diff
@@ -17739,3 +17739,36 @@ label monika_hedonism:
     m 7etd "It's no wonder most people don't have that belief...{w=0.3}it's too simple, where morality is complicated."
     m 1eud "So it makes sense why Oscar Wilde portrayed hedonism in a bad light."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_last_man",
+            category=["literature"],
+            prompt="The last man on Earth",
+			pool=True
+            random=False
+        )
+    )
+
```

Comment:
You'll want to put stories in `script-stories.rpy`, that's where we keep all our story topics.

Horror stories are in the second half of the file and have two labels that are called to set up the lights out and one to clean that up.